engine/include/cubos/engine/input/bindings.hpp

```diff
@@ -0,0 +1,34 @@
+/// @file
+/// @brief
+
+#pragma once
+
+#include <cubos/core/data/serializer.hpp>
+
+using cubos::core::data::Deserializer;
+using cubos::core::data::Serializer;
```

Comment:
Never do this in headers

Reply:
Posso manter o `using cubos::core::io::Key;`?